### ISSUE TITLE
Account balances are not synchronously updated

### ISSUE THREAD

**Issue by morelazers** [netgum/archanova#28]
When I receive a Completed AccountTransactionUpdated event, I expect to be able to access the Account's balance and receive the updated value.

Currently I have to set a 500ms timeout before accessing the balance else they do not contain the updated value(s).

**Comment by stanislaw-glogowski** [comment]
That's correct - we are not storing account real balance in our db - it's sync in background directly from json rpc `eth_getBalance` method. Balance check interval is set to 5 sec. so there could be some delay.

**Comment by stanislaw-glogowski** [comment]
I'll try to solve it with additional balance check after completed AccountTransactionUpdated event.
